docs/about.md

```diff
@@ -0,0 +1,28 @@
+<frontmatter>
+  title: "About"
+</frontmatter>
+
+# About
```

Comment:
My thought process was that from the developer guide, you might want to be able to access this information conveniently without having to go looking for it in other web pages - the acknowledgements aren't anywhere else on the dev guide, so I thought you might also want it here. You're right that it doesn't add any information that doesn't exist elsewhere. If it's unnecessary, I'll remove it.

Reply:
@kaixin-hc MarkBind  avoiding content duplication is supposedly one of the strong points of MarkBind. You might be able to have the content in the page (if useful) without duplicating the content in the source code.

Reply:
@damithc The issue here for is that the content is in the root README, but the MarkBind site rooted in the docs folder. While I think you could solve this by rooting the markbind site in the root of the teammates repository, I suspect that would be undesirable because the `_markbind` folder would then also be at the root, and that seems...disorganized. I am not aware of any other workaround for this :(